src/main/java/mods/railcraft/Railcraft.java

```diff
@@ -146,8 +194,22 @@ private void handleGatherData(GatherDataEvent event) {
     generator.addProvider(new RailcraftModelProvider(generator));
   }
 
+  @SubscribeEvent(priority = EventPriority.HIGHEST)
+  public void handleBiomeLoading(BiomeLoadingEvent event) {
+    if (!RailcraftConfig.common.enableOreGeneration.get()) {
+      return;
+    }
+    // BOILERPLATE! Will maybe turn into its own registry so we can iterate and such
+    if (event.getName() != null && RailcraftOrePlacements.TIN_ORE.bannedBiomesList().contains(event.getName().toString())) {
+      return;
+    }
+    event.getGeneration().addFeature(
+        GenerationStep.Decoration.UNDERGROUND_ORES,
+        RailcraftOrePlacements.TIN_ORE.placedFeature());
+  }
+
   @SubscribeEvent
-  public void handleWorldTick(TickEvent.WorldTickEvent event) {
```

Comment:
Add this back (maybe change your formatter settings if this was done by your IDE)